Products/ZenModel/ValueChangeThreshold.py

```diff
@@ -51,30 +52,31 @@ class ValueChangeThresholdInstance(MetricThresholdInstance):
     Threshold that emits an event when a value changes from its previous value. Does not send clear events.
     """
 
-    def __init__(self, id, context, dpNames, eventClass, severity):
-        MetricThresholdInstance.__init__(self, id, context, dpNames, eventClass, severity)
-        self._lastValues = {}
+    lastValues = Map.Locked(Map.Timed({}, 60*60*24)) # 24-hour timeout
```

Comment:
Please explain why this is now a class level variable.  It's not obvious why having an instance variable was not sufficient.